label_studio_ml/examples/tesseract/docker-compose.yml

```diff
@@ -12,15 +12,15 @@ services:
     volumes:
       - "./data/server:/data"
       - "./logs:/tmp"
-
+  
   minio:
     container_name: minio
-    image: quay.io/minio/minio
+    image: bitnami/minio:latest
     env_file:
       - example.env
     ports:
-      - 9000:9000
-      - 9001:9001
+      - 9003:9000
```

Comment:
is there any reason why did you change the defaults ports? if not - I'd suggest to keep them as before avoiding unnecessary changes